samples/kotlin-ks/build.gradle.kts

```diff
@@ -20,5 +20,6 @@ dependencies {
 
 configure<KtlintExtension> {
     verbose = true
-    reporter = ReporterType.JSON
+    outputToConsole = true
+    reporters = arrayOf(ReporterType.CHECKSTYLE, ReporterType.JSON)
```

Comment:
The extension should support doing `reporters  = listOf(ReporterType.CHECKSTYLE, ReporterType.JSON)`